### PR TITLE
Use common::setenv for portability to Windows

### PR DESCRIPTION
**Pull request by j-rivero**
Use the ignition::common setenv to be able to work on Windows. Also added missing std:: namespaces to getenv.

### CODE REVIEW

File: src/network/PeerTracker_TEST.cc [hunk 1]
```diff
@@ -281,11 +282,11 @@ TEST(PeerTracker, PartitionedEnv)
   ignition::common::Console::SetVerbosity(4);
   EventManager eventMgr;
 
-  setenv("IGN_PARTITION", "p1", 1);
+  ignition::common::setenv("IGN_PARTITION", "p1");
   auto tracker1 = PeerTracker(
       PeerInfo(NetworkRole::SimulationPrimary), &eventMgr);
 
-  setenv("IGN_PARTITION", "p2", 1);
+  ignition::common:setenv("IGN_PARTITION", "p2");
```
Comment (chapulina) [inline src/network/PeerTracker_TEST.cc:289]: There's something funky here that's messing up CI:

```
/var/lib/jenkins/workspace/ignition_gazebo-ci-pr_any-ubuntu_auto-amd64/ign-gazebo/src/network/PeerTracker_TEST.cc:289:19: error: expected primary-expression before ���:��� token
   ignition::common:setenv("IGN_PARTITION", "p2");
```

File: test/integration/save_world.cc [hunk 2]
```diff
@@ -27,6 +27,7 @@
 #include <sdf/World.hh>
 
 #include <ignition/common/Console.hh>
+#include <ignition/common/Util.hh>
```
Comment (chapulina) [inline test/integration/save_world.cc:30]: Nit: alphabetize

**Review by chapulina — COMMENTED** [review]
LGTM once CI is fixed.

File: test/integration/save_world.cc (hunk 2 above)
Comment (j-rivero) [reply test/integration/save_world.cc]: af72cc5

File: src/network/PeerTracker_TEST.cc (hunk 1 above)
Comment (j-rivero) [reply src/network/PeerTracker_TEST.cc]: 4dbb0ac

**Review by chapulina — APPROVED** [review]

**Pull request merged**